scipy/special/tests/test_mpmath.py

```diff
@@ -122,6 +122,30 @@ def test_hyp2f1_strange_points():
 
     FuncData(sc.hyp2f1, dataset, (0,1,2,3), 4, rtol=1e-10).check()
 
+@check_version(mpmath,'1.0.0')
+def test_hyp2f1_integer_c():
+    # a,b,c
+    pts = [
+        (1.35, 2.39, -2),
+        (1.20, 3.82, 0),
+        (-1, 3.2, -1),
+        (1.23, -4, -4),
+        (2.12, 0, -3),
+        (-2, 0.89, -4)
+        ]
+
+    # Test both the complex and real versions of hyp2f1
+    for z in (0.5+0.5j,0.5):
+        curr_pts=[p+(z,) for p in pts]
```

Comment:
Note that the Pep 8 checker is failing because of the lack of whitespace around the `=`. Remember to `pip install pep8` and run `pep8 scipy/special` to check style.

Reply:
In general, remember to include spaces between function arguments and around operators like `+`.

Reply:
@person142 Okay cool! I did not know about Pep 8...